src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java

```diff
@@ -33,7 +35,7 @@
         oneLineSummary = "Creates indices for Feature-containing files (eg VCF and BED files)",
```

Comment:
Add in the text `(Disabled) ` at the start of both the `summary` and `oneLineSummary` here.